imtiyaz/index.html

```diff
@@ -0,0 +1,301 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="UTF-8">
+	<title>CV Md. Imtiyaz Hossain</title>
+</head>
+<body>
+	<header>
+		<span style="margin-left: 20%;float: left;"><img src="./images/img.jpg" width="150px" height="160px"></span>
+		
+		<table style="text-align: right;margin-left: 51%;">
+			<tr>
+				<th>Md. Imtiyaz Hossain</th>
+			</tr>
+			<tr>
+				<td>Tejgaon 1/A Colony Bazar 9NO. Water Pump, Dhaka 1208</td>
+			</tr>
+			<tr>
+				<td>+8801682371979</td>
+			</tr>
+			<tr>
+				<td>imtiyaz.aust@gmail.com</td>
+			</tr>
+			<tr>
+				<td><a href="www.linkedin.com/in/ihhira" target="_blank">www.linkedin.com/in/ihhira</a></td>
```

Comment:
link not working, `href` value should be starting with `https://` (href="https://")